stylesheets/contributors.css

```diff
@@ -0,0 +1,68 @@
+.content {
+  position: absolute;
+  margin-top: 7%;
+  left: 50%;
+  -webkit-transform: translate(-50%, -50%);
```

Comment:
We need to support other browsers too...not just webkit browsers...if you're gonna use vendor-prefixes handle them all.